### ISSUE TITLE
Rule Change: check comma-separated keyframe selector also

### ISSUE THREAD

**Issue by Tanujkanti4441** [eslint/css#459]
### What rule do you want to change?

no-duplicate-keyframe-selectors

### What change do you want to make?

Generate more warnings

### How do you think the change should be implemented?

A new default behavior

### Example code

```css
@keyframes a {
  0%, 50% { opacity: 0; }
  50% { opacity: 1; }
}
```

### What does the rule currently do for this code?

It does not report `50%` as a duplicate keyframe selector.

### What will the rule do after it's changed?

It will report duplicate selectors in the `@keyframe` even if it is written as comma-separated selector.

### Participation

- [x] I am willing to submit a pull request to implement this change.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

**Comment by Pixel998** [comment]
Sounds good to me 👍. Marking as accepted.
